custom_components/hisense_tv/sensor.py

```diff
@@ -180,7 +180,7 @@ async def async_update(self):
         await mqtt.async_publish(
             hass=self._hass,
             topic=self._out_topic(
-                "/remoteapp/tv/platform_service/%s/actions/picturesetting"
```

Comment:
`picturesetting` and `gettvstate` are completely different topics. If you remove `picturesetting`, the sensor won't show data unless TV sends it on its own (e.g. after changing some settings via the onscreen menu).

I'm also not sure if `gettvstate` has an action `get_menu_info` - see the payload in line183. Can you provide some information on that?